client/checkout/platform-checkout/style.scss

```
@@ -66,7 +66,7 @@
 		&::after {
 			content: '';
 			position: absolute;
-			left: calc( 100% + 15px );
+			left: calc( min(calc(100% + 30px), calc( 85vw - max(85vw - 100% - 94px, 300px) ) ) - 15px);
```

Looks like the linter is upset about this line - I think because of the `calc` inside of the `min()`.

```suggestion
			left: calc( min( 100% + 30px, 85vw - max(85vw - 100% - 94px, 300px) ) - 15px)
```

---

Looks like I missed a semi-colon on my suggested change. I pushed a commit to fix that and the CSS formatter added a few changes based on line length on commit.